CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+#Contributing
+
+### Setup
+- Fork the repository recursively to get the submodules ```git clone --recursive git@github.com:MSFTOSSMgmt/bld-omsagent.git```
```

Comment:
The 'git clone' is a long line. Suggest adding ":" after submodules, then go to next line for the command.

Otherwise fine, signed off.